data/json/items/ammo/50pellet.json

```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "id": "50pellet",
+    "type": "AMMO",
+    "name": ".50 Pellet",
+    "description": "A huge .50 pellet for big bore air rifles. It causes tremendous tissue damage for an air rifle.",
+    "weight": 10,
+    "volume": 1,
```

Comment:
250 ml is a really high volume for a pellet.

Reply:
Didn't realize those ammo files were already compatible with the new units system.